src/main/java/edu/hm/hafner/coverage/FileNode.java

```diff
@@ -193,16 +243,44 @@ private void mergeCounters(final FileNode otherFile) {
             addValue(branchValue);
         }
 
-        otherFile.getValues().stream()
-                .filter(value -> value.getMetric() == Metric.COMPLEXITY)
-                .forEach(this::addValue);
+        var mcdcPairValue = mcdcPairCoverage.build();
+        if (mcdcPairValue.isSet()) {
+            addValue(mcdcPairValue);
+        }
+
+        var functionCallValue = functionCallCoverage.build();
+        if (functionCallValue.isSet()) {
+            addValue(functionCallValue);
+        }
+    }
+            
+    void mergeLeftRight(final int line, final int leftCovered, final int leftMissed, final int rightCovered, final int rightMissed, 
```

Comment:
When not used somewhere else:
```suggestion
    private void mergeLeftRight(final int line, final int leftCovered, final int leftMissed, final int rightCovered, final int rightMissed, 
```

Reply:
Complete